tests/jax/components/executing/base_test.py

```diff
@@ -67,18 +61,6 @@ def mock_executor() -> Executor:
     return executor
 
 
-def test_on_building_init(mock_builder: Builder) -> None:
-    """Test on_building_init method from ExecutorInit
-
-    Args:
-        mock_builder:Builder
-    """
-    executor_init = ExecutorInit()
-    executor_init.on_building_init(builder=mock_builder)
-
-    assert mock_builder.store.networks == "after_network_factory"
```

Comment:
Why was this test removed?

Reply:
See `ExecutorInit`. It now no longer creates the networks since they are just created in `TrainerInit`

Reply:
Make sense. Thank you